temporal-sdk/src/main/java/io/temporal/internal/client/RootWorkflowClientInvoker.java

```diff
@@ -49,6 +49,7 @@
 
 public class RootWorkflowClientInvoker implements WorkflowClientCallsInterceptor {
   private static final Logger log = LoggerFactory.getLogger(RootWorkflowClientInvoker.class);
+  private static final long POLL_UPDATE_TIMEOUT_S = 60l;
```

Comment:
Why 601?

Reply:
that is an `l`, let me change it to caps.